code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,320 @@
+	name = "destabilized extract"
+	desc = "It seems ready to burst at any moment."
+	effect = "destabilized"
+	icon_state = "destabilized"
+
+/obj/item/slimecross/destabilized/throw_impact(atom/A, var/datum/thrownthing/T)
+	if(!..()) //For those tactical catches
+		if(hitEffect(A, T.thrower))
+			playsound(get_turf(A), 'sound/misc/splort.ogg', get_volume_by_throwforce_and_or_w_class(), 1, -1)
+			qdel(src)
+
+/obj/item/slimecross/destabilized/proc/hitEffect(atom/A, mob/thrower)
+	return TRUE //But nothing happens...
+
+/obj/item/slimecross/destabilized/grey
+	colour = "grey"
+
+/obj/item/slimecross/destabilized/grey/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/mob/living/simple_animal/slime/random/S = new (get_turf(A))
+	S.rabid = TRUE
+	S.Friends += thrower
+	if(isliving(A))
+		var/mob/living/L = A
+		S.Target = L
+		if(S.CanFeedon(L))
+			S.Feedon(L)
+
+/obj/item/slimecross/destabilized/orange
+	colour = "orange"
+
+/obj/item/slimecross/destabilized/orange/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		if(L.fire_stacks < 1)
+			L.fire_stacks = 1
+		L.IgniteMob()
+	else
+		new /obj/effect/hotspot(get_turf(A))
+
+/obj/item/slimecross/destabilized/purple
+	colour = "purple"
+
+/obj/item/slimecross/destabilized/purple/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.reagents.add_reagent("regen_jelly", 50)
+	else
+		var/datum/reagents/R = new/datum/reagents(50)
+		R.add_reagent("regen_jelly", 50)
+		var/datum/effect_system/smoke_spread/chem/smoke = new
+		smoke.set_up(R, 1, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/blue
+	colour = "blue"
+
+/obj/item/slimecross/destabilized/blue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/datum/reagents/R = new/datum/reagents(50)
+	R.add_reagent("water", 50)
+	R.reaction(get_turf(A), TOUCH, 50)
+	qdel(R) //Reaction doesnt use up the reagents
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		C.slip(100, src, FALSE, 20, FALSE)
+
+/obj/item/slimecross/destabilized/metal
+	colour = "metal"
+
+/obj/item/slimecross/destabilized/metal/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		to_chat(L, "<span class='warning'>You suddenly find yourself trapped in a dark cage!</span>")
+		L.apply_status_effect(/datum/status_effect/cageTrapped)
+	else
+		new /obj/structure/metalCage(get_turf(A))
+
+/obj/item/slimecross/destabilized/yellow
+	colour = "yellow"
+
+/obj/item/slimecross/destabilized/yellow/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.electrocute_act(rand(5,20), src, 1, 1)
+		playsound(L, "sparks", 50, 1)
+	else
+		empulse(get_turf(A), 1, 1)
+
+/obj/item/slimecross/destabilized/darkpurple
+	colour = "dark purple"
+
+/obj/item/slimecross/destabilized/darkpurple/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.fire_stacks += 5
+		L.reagents.add_reagent("plasma", 30)
```

Comment:
this is a death sentence

Reply:
Not really, it's survivable if you have charcoal on hand, which most medbays have readily available.

Reply:
if it requires victims to be in medbay to survive a single hit it's pretty strong